### PR TITLE
@counter-style: made the example a live example

### PR DESCRIPTION
**Pull request by estelle**
has been supported for years. might as well show it.

### CODE REVIEW

File: files/en-us/web/css/@counter-style/index.md [hunk 1]
```diff
@@ -100,20 +100,26 @@ The above counter style rule can be applied to lists like this:
...
-Ⓓ Four
-Ⓔ Five
-…
-Ⓨ Twenty-five
-Ⓩ Twenty-six
-27 Twenty-seven
-28 Twenty-eight
-29 Twenty-nine
-30 Thirty
+Which will produce a lists like this:
```
Comment (bsmth) [inline files/en-us/web/css/@counter-style/index.md:103]:
```suggestion
Which will produce a list like this:
```

File: files/en-us/web/css/@counter-style/index.md [hunk 2]
```diff
@@ -100,20 +100,26 @@ The above counter style rule can be applied to lists like this:
...
+<p>...</p>
+<ol class="items" start="25">
+  <li>twenty-five</li>
+  <li>twenty-six</li>
+  <li>twenty-seven</li>
+  <li>twenty-eight</li>
+</ol>
+```
+
+{{EmbedLiveSample('Specifying symbols with counter-style')}}
```
Comment (bsmth) [inline files/en-us/web/css/@counter-style/index.md:122]: There's a vertical scrollbar in the example, do you want to add a height param to get rid of it? Fine either way, I think.

File: files/en-us/web/css/@counter-style/index.md (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/css/@counter-style/index.md:103]: I think it reads a bit strangely to have both of the previous lead-in sentences ending "...lists like this". Maybe change this one to something like

> The above code produces the following result:

?

**Review by chrisdavidmills — APPROVED** [review]
@estelle LGTM, pending the items @bsmth commented on.

File: files/en-us/web/css/@counter-style/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/css/@counter-style/index.md:103]:
```suggestion
The above code produces the following result:
```

File: files/en-us/web/css/@counter-style/index.md (hunk 2 above)
Comment (estelle) [inline files/en-us/web/css/@counter-style/index.md:122]:
```suggestion
{{EmbedLiveSample('Specifying symbols with counter-style', '', '300')}}
```

**Pull request merged**